src/docs/markdown/caddyfile/directives/file_server.md

```diff
@@ -69,6 +70,9 @@ file_server [<matcher>] [browse] {
 
 - **pass_thru** <span id="pass_thru"/> enables pass-thru mode, which continues to the next HTTP handler in the route if the requested file is not found, instead of triggering a `404` error (invoking [`handle_errors`](handle_errors) routes). Practically, this is only useful inside of a [`route`](route) block with other handler directives following `file_server`, because this directive is effectively [ordered last](/docs/caddyfile/directives#directive-order).
 
+- **sort** <span id="sort"/> specifies how to sort directory listings. The options are sortBy (`name`, `namedirfirst`, `size`, `time`) and order (`asc` or `desc`) separated by spaces. The first option must be `sortBy`, and the second option must be `order` (if exists). For example, `sort name desc` will sort by name in descending order.
```

Comment:
```suggestion
- **sort** <span id="sort"/> changes the default sort for directory listings. The first parameter is the field/column to sort by: `name`, `namedirfirst`, `size`, or `time`. The second argument is an optional direction: `asc` or `desc`. For example, `sort name desc` will sort by name in descending order.
```

Reply:
This also needs to be moved up into the sub-list of browse.

Reply:
Moved to `browse`, but the location of `sort` config...

It seems I need to change the caddy code again.

```
file_server browse {
	sort size desc
}
```

```
file_server {
	browse {
		sort size desc
	}
}
```
Change 1 to 2? or make both valid, but 2 has higher priority?

Reply:
Good question; I think the first one is not really a thing; i.e. it's just a shorthand for the default browse (no block).

So aim for the second one. Just put it inside the existing browse block. :+1: